library/src/nvidia_detail/hipblas.cpp

```diff
@@ -12582,6 +12633,30 @@ catch(...)
     return exception_to_hipblas_status();
 }
 
+hipblasStatus_t hipblasScalEx_v2(hipblasHandle_t handle,
+                                 int             n,
+                                 const void*     alpha,
+                                 hipDataType     alphaType,
+                                 void*           x,
+                                 hipDataType     xType,
+                                 int             incx,
+                                 hipDataType     executionType)
+try
+{
+    return hipCUBLASStatusToHIPStatus(cublasScalEx((cublasHandle_t)handle,
+                                                   n,
+                                                   alpha,
+                                                   HIPDatatypeToCudaDatatype_v2(alphaType),
+                                                   x,
+                                                   HIPDatatypeToCudaDatatype_v2(xType),
+                                                   incx,
+                                                   HIPDatatypeToCudaDatatype_v2(executionType)));
+}
+catch(...)
+{
+    return exception_to_hipblas_status();
+}
```

Comment:
I guess you don't want to try my macro ILP64 strategy here :-)   I guess not if you can delete one soon.